Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR_test.py

```diff
@@ -1580,3 +1581,91 @@ def test_get_xsoar_close_reasons(mocker):
     }
     mocker.patch.object(demisto, 'internalHttpRequest', return_value=mock_response)
     assert get_xsoar_close_reasons() == list(XSOAR_RESOLVED_STATUS_TO_XDR.keys()) + ['CustomReason1', 'CustomReason 2', 'Foo']
+
+
+@freeze_time('1970-01-01 00:00:00.100')
+def test_fetch_incidents_dedup():
+    """
+    Unit test to verify that incidents that occur in the same instant are not not missed or duplicated.
```

Comment:
Change the docstring to be more informative - add cases.

Reply:
Will do